templates/cicd/pipeline.yml

```
@@ -20,5 +20,7 @@ source:
 # to your environments.
 stages:{{range .Stages}}
     - # The name of the environment to deploy to.
-      name: {{.Name}}{{end}}
+      name: {{.Name}}
+      # The commands to run a set of tests for your service in this stage.
+      test_commands: {{.TestCommands}}{{end}}
```

My assumption is that we want `test_commands` to be optional, so we should only render it if the slice is not nil:
```suggestion
      name: {{.Name}}
      {{- if .TestCommands}}
      test_commands: 
      {{- range .TestCommands}}
      - {{.}}
      {{- end}}{{end}}{{end}}
```

---

Note: offline, I suggested a compromise solution: The test_commands field will remain, for the sake of discoverability, but commented out by default.